nix/overlay.nix

```diff
@@ -13,6 +13,8 @@ let
 in
 self: super: {
   sourcer = super.callPackage ./lib/sourcer.nix { };
+  extensions = super.callPackage ./pkgs/extensions { inherit allInOne incremental static tag rustFlags; };
```

Comment:
hmm this isn't the extensions repo

Reply:
Does this have to be matching repo name? openebs ?

Reply:
Done. made it openebs now